dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1269,6 +1271,9 @@ def test_filters(tmpdir, write_engine, read_engine):
 def test_filters_v0(tmpdir, write_engine, read_engine):
     if write_engine == "fastparquet" or read_engine == "fastparquet":
         pytest.importorskip("fastparquet", minversion="0.3.1")
+
+    if read_engine == "pyarrow" and pa.__version__ >= LooseVersion("1.0.0"):
+        pytest.skip("Filtering is more thorough (row-wise) in PyArrow>=1.0.0")
```

Comment:
I don't understand this skip message. Does something need to be updated in the test, or is it not applicable?

Reply:
Presumably you could phrase this as "the results will be the same for older pyarrow and fastparquet"

Reply:
The new dataset API in pyarrow will actually filter the data (so you get back a single row with "aa") before returning, while the current version in dask only filters on the row group level (in which case here nothing is filtered, as other values are present in the row group, and the resulting dask dataframe is identical to the original `ddf`).

Reply:
So something like `assert_eq(ddf2, ddf[ddf["at"] == "aa"])` would hold, instead of `assert_eq(ddf2, ddf)`, to say it in code

Reply:
Thanks for the feedback - I added a comment to the code, and am no longer skipping the test.  I took Joris' suggestion,  and changed the problematic assertion to `assert_eq(ddf2, ddf[ddf["at"] == "aa"], check_index=False)`.